Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Beckn/ACL/Confirm.hs

```diff
@@ -50,6 +50,7 @@ buildConfirmReqV2 req isValueAddNP = do
   let nightSafetyCheck = fulfillment.fulfillmentCustomer >>= (.customerPerson) >>= (.personTags) & getNightSafetyCheckTag isValueAddNP
       enableFrequentLocationUpdates = fulfillment.fulfillmentCustomer >>= (.customerPerson) >>= (.personTags) & getEnableFrequentLocationUpdatesTag
   toAddress <- fulfillment.fulfillmentStops >>= Utils.getDropLocation >>= (.stopLocation) & maybe (pure Nothing) Utils.parseAddress
+  estimateId <- req.confirmReqMessage.confirmReqMessageOrder.orderItems >>= listToMaybe >>= (.itemId) & fromMaybeM (InvalidRequest "itemId not found")
```

Comment:
backward compatibility issue, we can fallback to the value where it was previously ?

Reply:
Have added estimateId to booking table, will do same to handle for rider-app too

Reply:
but this will cause issue if driver app is released first, right ?

Reply:
No because bap will send us back the same item.id that we send to them